File: wfm-develop mobile 07.05.2026/wfm/mobile/android/app/src/main/java/com/beyondviolet/wfm/core/models/Hint.kt
package com.beyondviolet.wfm.core.models

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Hint(
    val id: Int,
    val text: String,
    @SerialName("work_type_id") val workTypeId: Int? = null,
    @SerialName("zone_id") val zoneId: Int? = null
)

/** Обёртка ответа GET /tasks/hints → { "hints": [...] } */
@Serializable
data class HintsResponse(val hints: List<Hint>)

File: wfm-develop mobile 07.05.2026/wfm/mobile/ios/WFMApp/Core/Models/Hint.swift
import Foundation

struct Hint: Identifiable, Codable, Hashable {
    let id: Int
    let text: String
    let workTypeId: Int?
    let zoneId: Int?

    enum CodingKeys: String, CodingKey {
        case id
        case text
        case workTypeId = "work_type_id"
        case zoneId = "zone_id"
    }
}

/// Обёртка ответа GET /tasks/hints → { "hints": [...] }
struct HintsResponse: Codable {
    let hints: [Hint]
}

